src/moveit_opw_kinematics_plugin.cpp

```diff
@@ -84,32 +84,28 @@ bool MoveItOPWKinematicsPlugin::initialize(const moveit::core::RobotModel& robot
     ik_group_info_.link_names.push_back(tip_frames_[i]);
   }
 
-  // Choose what ROS service to send IK requests to
-  ROS_DEBUG_STREAM_NAMED("opw", "Looking for ROS service name on rosparam server with param: "
-                                    << "/kinematics_solver_service_name");
-  std::string ik_service_name;
```

Comment:
`ik_service_name` did not appear to be used anywhere.

Reply:
Jup, that's a copy past leftover from the `srv_kinematics_plugin`. No other IK plugin in MoveIt seems to do this. (I think the move group interface also creates this service, I'm not sure.)